src/components/Support/index.tsx

```diff
@@ -1,137 +1,219 @@
+        <span>resource needs</span>.
+      </>
+    ]
+  },
+  {
+    icon: TargetIcon,
+    title: 'Best practices and standards',
+    points: [
+      <>
+        <span>Architect</span> solutions for scalability.
+      </>,
+      <>
+        <span>Develop</span> common standards and frameworks.
+      </>,
+      <>
+        <span>Assist</span> with prototype development.
+      </>
+    ]
+  }
+]
+
+const colors = ['text-purple', 'text-blue', 'text-orange']
+
+const getAccentColor = (index: number) => {
+  return colors[index % colors.length]
+}
+
+const getSpanAccentColor = (index: number) => {
+  const spanColors = [
+    '[&>span:nth-child(3n-2)]:text-purple [&>span:nth-child(3n-1)]:text-blue [&>span:nth-child(3n)]:text-orange',
+    '[&>span:nth-child(3n-2)]:text-blue [&>span:nth-child(3n-1)]:text-orange [&>span:nth-child(3n)]:text-purple',
+    '[&>span:nth-child(3n-2)]:text-orange [&>span:nth-child(3n-1)]:text-purple [&>span:nth-child(3n)]:text-blue'
+  ]
+  return spanColors[index % colors.length]
+}
+
+const SupportPage: React.FC = () => {
+  const [openDialog, setOpenDialog] = useState(false)
+  return (
+    <>
+      <PageContent>
+        <HeroContainer className={styles.supportHero}>
+          <h1
+            className={cn(
+              styles.heading,
+              'mx-auto',
+              'font-extrabold text-5xl text-center'
+            )}
+          >
+            Platinum Services for ML Operations
+          </h1>
+          <p
+            className={cn(
+              'text-center',
+              'text-xl',
+              'mt-5',
+              'mx-auto',
+              'max-w-2xl',
+              'text-gray-500'
+            )}
+          >
+            Accelerate your business with our world-class data engineering, data
+            science, and project management experts and deliver rapid success in
+            your ML projects.
+          </p>
+
+          <button
+            onClick={() => setOpenDialog(true)}
+            className={cn(
+              'mt-10',
+              'mx-auto',
+              'block',
+              'px-6',
+              'py-2',
+              'rounded-md',
+              'text-lg',
+              'font-semibold',
+              'text-blue',
+              'border-2',
+              'border-blue',
+              'hover:bg-blue',
+              'hover:text-white'
+            )}
+          >
+            Request a quote
+          </button>
+        </HeroContainer>
+        <LayoutWidthContainer>
+          <div className="max-w-2xl mx-auto text-center">
+            <blockquote className="mt-14 italic text-lg">
+              <p className="before:content-['\201C'] after:content-['\201D']">
+                I can confidently say that DVC AI&apos;s Platinum Engineering
+                Services has been an invaluable asset to our team. Their expert
+                knowledge and hands-on keyboard support enabled us to quickly
```

Comment:
```suggestion
                knowledge and hands-on custom support enabled us to quickly
```

Reply:
Per discussion on messaging of DVC Enterprise Services at today's GTM meeting, let's change this to "custom." cc: @joshcohen15 @dmpetrov